sdk/ml/azure-ai-ml/tests/workspace/unittests/test_workspace_operations.py

```diff
@@ -31,13 +31,13 @@ def mock_credential() -> Mock:
 @pytest.fixture
 def mock_workspace_operation(
     mock_workspace_scope: OperationScope,
-    mock_aml_services_2022_12_01_preview: Mock,
+    mock_aml_services_2023_02_01_preview: Mock,
```

Comment:
Also can you change the two in this file as well to be mock_aml_services_2022_12_01_preview